git_machete/utils.py

```diff
@@ -328,3 +329,53 @@ class CommandResult(NamedTuple):
     stdout: str
     stderr: str
     exit_code: int
+
+
+class Qualifiers:
+    rebase: Optional[bool]
+    merge: Optional[bool]
+    push: Optional[bool]
+
+    def __init__(self, annotation: str):
+        self.__annotation = annotation
+        self.__rebase_text = ''
+        self.__push_text = ''
+        self.__merge_text = ''
+        self.rebase = None
+        self.push = None
+        self.merge = None
+
+        rebase_match = re.match('.*rebase=no.*', annotation)
```

Comment:
The qualifier should be delimited by whitespace (or text start/end) on both sides (as in, `push=norebase=no` shoudln't match)